util/threadpool_imp.cc

```diff
@@ -110,6 +134,13 @@ struct ThreadPoolImpl::Impl {
 
  int total_threads_limit_;
  std::atomic_uint queue_len_;  // Queue length. Used for stats reporting
+ // Number of reserved threads, managed by ReserveThreads(..) and
+ // ReleaseThreads(..), if a thread id is beyond total_threads_limit_ -
+ // reserved_threads_, its thread will be blocked to ensure the reservation
+ // mechanism
```

Comment:
Comments needed update as now we are not doing "if a thread id is beyond total_threads_limit_ // reserved_threads_,"?

Reply:
Sure. Will update it.